x/crosschain/client/cli/cli_tss.go

```diff
@@ -41,6 +41,32 @@ func CmdShowTSS() *cobra.Command {
 	return cmd
 }
 
+func CmdListTssHistory() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-tss-history",
```

Comment:
Maybe we could rename the query cmd into `TssHistory` instead of `ListTssHistory`

Reply:
We have been using `show-XXX` and `list-XXX` for all items to show query single and multiple . 
This goes with that nomenclature